docs/tools/rmmod_tb.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
I feel like using `sh` instead of a specific shell here caused problems in the past

Reply:
Should I use bash?

Reply:
Yes, but check that it doesn't break it. See https://canary.discord.com/channels/595304521857630254/595304521857630259/811371377398382632

Reply:
a19eab4